src/journal-remote/journal-remote-main.c

```diff
@@ -152,6 +155,27 @@ static int dispatch_http_event(sd_event_source *event,
                                uint32_t revents,
                                void *userdata);
 
+static int build_accept_encoding(char **ret) {
+        float q = 1.0;
+
+        assert(ret);
+
+        if (arg_compression.size <= 0)
+                return 0;
+
+        float step = q / arg_compression.size;
+        _cleanup_free_ char* buf = NULL;
+
```

Comment:
- Spurious empty line after declaration. 
- Please merge the declaration of `q` in the above here.
- `char* buf` -> `char *buf`
```
float q = 1.0, step = 1.0 / arg_compression.size;
_cleanup_free_ char *buf = NULL;
```